src/arrayReverse.test.js

```diff
@@ -8,13 +8,36 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(arrayReverse([])).toBeInstanceOf(Array);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([])).toEqual([]);
+  });
+
+  it(`should accept cyrillic letters`, () => {
+    expect(arrayReverse(['привет', 'друг'])).toEqual(['гурдте', 'вирп']);
+  });
```

Comment:
This test is not needed because it is said in the description that it can contain Latin letters, numbers and special symbols (not Cyrillic)

Reply:
I tried to write a test that Cyrillic characters are not acceptable, but they are so I changed it